src/main/java/dk/kb/discover/SolrService.java

```diff
@@ -280,6 +314,35 @@ public String query(String q, List<String> fq, Integer rows, Integer start, Stri
         if (facetField != null) {
             facetField.forEach(ff -> builder.queryParam(FACET_FIELD, ff));
         }
+        
+        /*
+        //All the spellcheck parameters
+        if (spellcheck != null) {
+        	   builder.queryParam(SPELLCHECK, Boolean.parseBoolean(spellcheck));
+        }
+        
+        if (spellcheckBuild != null) {
+        	   builder.queryParam(SPELLCHECK_BUILD, Boolean.parseBoolean(spellcheckBuild));
+        }
+        
+        if (spellcheckReload != null) {
+     	   builder.queryParam(SPELLCHECK_RELOAD, Boolean.parseBoolean(spellcheckReload));
+        }
+        */
```

Comment:
Why this commented out code? Is it lefftover from experimenting?

Reply:
Correct, from the bug hunt. Is removed now.